Add no seed installation manual (#9349)
decidim/decidim: docs/modules/install/pages/noseed.adoc

= Install with no seed

> You may want to setup your decidim with no seeds.

Reviewer: ```suggestion
You may want to setup your decidim with an empty database.
```

In this case, please follow the steps bellow:

. Follow steps 1 through 5 of previous section(xref:install:manual.adoc[Manual Installation] section).

> 

Reviewer: I didn't try it, but as far as I know, if you remove the white lines then you can drop the "`[start=3]`" and also the hardcoded numbers (like the "3." and "4.")

Author: You are right about the numbering before the ordered list, but '[start=n]' seems to be essential in this case.

Reviewer: I've just seen it in other changes that I was working on, you're absolutely right. How it is now it's perfect!

> . Since you don't need to add the seeds, you are good to go with only running the migrations. run the migrations from the root of your project in the terminal

Reviewer: ```suggestion
. Run the migrations from the root of your project in the terminal
```
I think we can skip the first phrase

[source,bash]
----
bin/rails db:create db:migrate
----

[start=3]
3. Now you need to create your admin user, in order to be able to login to the system. in the terminal type the following:

[source,bash]
----
bundle exec rails s
----

[start=4]
4. Once you entered the console, copy and paste the following:

[source,bash]
----
  admin = Decidim::User.find_or_initialize_by(email: "admin@example.org")

  admin.update!(
    name: Faker::Name.name,
    nickname: Faker::Twitter.unique.screen_name,
    password: "decidim123456",
    password_confirmation: "decidim123456",
    organization: organization,
    confirmed_at: Time.current,
    locale: I18n.default_locale,
    admin: true,
    tos_agreement: true,
    personal_url: Faker::Internet.url,
    about: Faker::Lorem.paragraph(sentence_count: 2),
    accepted_tos_version: organization.tos_version,
    admin_terms_accepted_at: Time.current
  )

> ----

Reviewer: Since https://github.com/decidim/decidim/pull/8010 we have a rake task for this:
 
```bash
$ bin/rails decidim_system:create_admin
Please, provide the following attributes to create a new system admin (currently there are existing system admins):
email: admin@example.org
Password: 
Password confirmation: 
System admin created successfully
```

It seems we forgot to update the docs about this, but we should change it to this as is more sysadmin friendly for sure.
